web_src/js/features/repo-commit.ts

```diff
@@ -24,3 +24,20 @@ export function initCommitStatuses() {
     });
   });
 }
+
+export function initCommitFileHistoryFollowRename() {
+  const checkbox : HTMLInputElement | null = document.querySelector('input[name=history-enable-follow-renames]');
+
+  if (!checkbox) {
+    return;
+  }
+  const url = new URL(window.location.toString());
+  checkbox.checked = url.searchParams.has('history_follow_rename', 'true');
+
+  checkbox.addEventListener('change', () => {
+    const url = new URL(location.toString());
```

Comment:
```suggestion
    const url = new URL(window.location);
```

Url constructor implicitely calls `toString`. And I prefer to use `window.location` over `location` global.

Reply:
@silverwind I added .toString() as the vscode TypeScript extension didn't like it (said there was a type mismatch with window.location not being a string), but if that's ok for you I'm changing this.

Reply:
The compliance check gave me the exact same error as the one I had in vscode.